mnefun/_mnefun.py

```diff
@@ -822,38 +863,141 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
         raise IOError('input file not found: %s' % fname_in)
     if not op.isdir(op.dirname(op.abspath(fname_out))):
         raise IOError('output directory for output file does not exist')
-    fnames_in = [fname_in]
-    for ii in range(1, 11):
-        next_name = op.splitext(fname_in)[0] + '-%s' % ii + '.fif'
-        if op.isfile(next_name):
-            fnames_in.append(next_name)
-        else:
-            break
```

Comment:
The new version of the code is working for me so far

Jason D. Yeatman, PhD
Assistant Professor, Institute for Learning & Brain Sciences (I-LABS)
Department of Speech & Hearing Sciences
University of Washington
http://depts.washington.edu/bdelab/

On Tue, Feb 16, 2016 at 1:14 PM, Eric Larson <notifications@github.com>
wrote:

> In mnefun/_mnefun.py
> <https://github.com/LABSN/mnefun/pull/112#discussion_r53079378>:
>
> > @@ -822,38 +863,141 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
> >          raise IOError('input file not found: %s' % fname_in)
> >      if not op.isdir(op.dirname(op.abspath(fname_out))):
> >          raise IOError('output directory for output file does not exist')
> > -    fnames_in = [fname_in]
> > -    for ii in range(1, 11):
> > -        next_name = op.splitext(fname_in)[0] + '-%s' % ii + '.fif'
> > -        if op.isfile(next_name):
> > -            fnames_in.append(next_name)
> > -        else:
> > -            break
>
> @kambysese <https://github.com/kambysese> this function is used outside
> of the context of analysis scripts though. What it was supposed to do
> (before these changes) was take a given input file name, see if any -1
> files exist, and automatically *also* process those with MaxFilter to get
> the positions. Make sense now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LABSN/mnefun/pull/112/files#r53079378>.
>